qa/L0_infer/infer_test.py

```diff
@@ -39,6 +39,11 @@
 TEST_SYSTEM_SHARED_MEMORY = bool(int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
 TEST_CUDA_SHARED_MEMORY = bool(int(os.environ.get('TEST_CUDA_SHARED_MEMORY', 0)))
 CPU_ONLY = (os.environ.get('TENSORRT_SERVER_CPU_ONLY') is not None)
+BACKENDS = os.environ.get('BACKENDS', "graphdef savedmodel netdef onnx libtorch plan custom")
+ENSEMBLES = bool(int(os.environ.get('ENSEMBLES', 1)))
+
+if ENSEMBLES:
+    BACKENDS="graphdef savedmodel netdef onnx libtorch plan custom"
```

Comment:
Don't override `BACKENDS`, in that way we have finer control over "ensemble" x "backend" combinations.

Reply:
Done